worlds/witness/options.py

```diff
@@ -210,50 +248,60 @@ class TrapWeights(OptionDict):
 
 
 class PuzzleSkipAmount(Range):
-    """Adds this number of Puzzle Skips into the pool, if there is room. Puzzle Skips let you skip one panel.
-    Works on most panels in the game - The only big exception is The Challenge."""
+    """
+    Adds this number of Puzzle Skips into the pool, if there is room. Puzzle Skips let you skip one panel.

```

Comment:
idk, could be "many", maybe not. There's a lot of ways to word this, how it is now is fine though
```suggestion
    Adds this many Puzzle Skips into the pool, if there is room. Puzzle Skips let you skip one panel.
```